### PR TITLE
Add "Export CSV" button to the Dashboard (CU-1mek79g)

### PR DESCRIPTION
**Pull request by trisapeace**
I just copied this functionality from Buttons and tweaked it to work for Sensors.

While testing, I discovered that I could download the CSV Export using a cookie from the browser even after I'd logged out using the "Log Out" button. So the `req.session.destroy()` call fixes that issue. The same thing came up in Buttons, so it's fixed in this PR: https://github.com/bravetechnologycoop/BraveButtons/pull/130 

## Test Plan
- :heavy_check_mark: Deploy on `staging.sensors.brave.coop`
- :heavy_check_mark:  Show a copy of the results to Gordon and Oona and see what they think
- :heavy_check_mark: Run smoketests
- :heavy_check_mark: Make sure you can't download the CSV unless logged in
- :heavy_check_mark: Pushing the "Export CSV" button downloads a CSV file with the current datetime in the name
- :heavy_check_mark: Data in fields comes from the right rows in the DB
- :heavy_check_mark:  A variety of data in the fields
   - :heavy_check_mark:  Interesting dates (morning, afternoon, midnight, noon, leap year, ...)
   - :heavy_check_mark: Special characters (accents, unicode, commas, quotes, ...)
- :heavy_check_mark: Resulting CSV can be imported into Google Sheets and can use the Google Sheets filters and Dates

### CODE REVIEW

**Comment by trisapeace** [comment]
@mariocimet @schwarrrtz 
I'm still waiting to hear back from Gordon and/or Oona whether this set of data will be sufficient for their reporting needs. So it can be looked at now or you can wait until I have that sorted out with them. This code is currently deployed on `https://staging.sensors.brave.coop`.

File: server/dashboard.js [hunk 1]
```diff
@@ -121,13 +124,41 @@ async function renderLoginPage(req, res) {
...
+    'Radar Type',
+    'Active?',
+    'Session ID',
+    'Session Start',
+    'Session Responded At',
+    'Last Session Activity',
+    'Session Incident Type',
+    'Session State',
+    'Alert Type',
+  ]
```
Comment (trisapeace) [inline server/dashboard.js:151]: Note to self: Confirm these fields with Gordon and/or Oona.

**Comment by trisapeace** [comment]
Gordon says that this looks good. Oona didn't know whether the data would help her or not. Instead, she told me the sorts of graphs that she'll eventually want, and I believe that that can be achieved with this data. So this is now ready for review @mariocimet and @schwarrrtz .

File: server/dashboard.js (hunk 1 above)
Comment (trisapeace) [reply server/dashboard.js:151]: Done.

**Review by mariocimet — APPROVED** [review]

**Pull request merged**